app/src/organisms/QuickTransferFlow/QuickTransferAdvancedSettings/FlowRate.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react'
+import { createPortal } from 'react-dom'
+import { useTranslation } from 'react-i18next'
+import {
+  Flex,
+  SPACING,
+  DIRECTION_COLUMN,
+  ALIGN_CENTER,
+  POSITION_FIXED,
+  COLORS,
+} from '@opentrons/components'
+import type { SupportedTip } from '@opentrons/shared-data'
+import { getTipTypeFromTipRackDefinition } from '@opentrons/shared-data'
+
+import { getTopPortalEl } from '../../../App/portal'
+import { ChildNavigation } from '../../ChildNavigation'
+import { InputField } from '../../../atoms/InputField'
+import { NumericalKeyboard } from '../../../atoms/SoftwareKeyboard'
+
+import { ACTIONS } from '../constants'
+import type {
+  QuickTransferSummaryState,
+  QuickTransferSummaryAction,
+  FlowRateKind,
+} from '../types'
+
+interface FlowRateEntryProps {
+  onBack: () => void
+  state: QuickTransferSummaryState
+  dispatch: React.Dispatch<QuickTransferSummaryAction>
+  kind: FlowRateKind
+}
+
+export function FlowRateEntry(props: FlowRateEntryProps): JSX.Element {
+  const { onBack, state, dispatch, kind } = props
+  const { i18n, t } = useTranslation(['quick_transfer', 'shared'])
+  const keyboardRef = React.useRef(null)
+
+  const [flowRate, setFlowRate] = React.useState<number | null>(
+    kind === 'aspirate'
+      ? state.aspirateFlowRate
+      : kind === 'dispense'
+      ? state.dispenseFlowRate
+      : null
+  )
+
+  // use lowVolume for volumes lower than 5ml
+  const tipType = getTipTypeFromTipRackDefinition(state.tipRack)
+  const flowRatesForSupportedTip: SupportedTip | undefined =
+    state.volume < 5
+      ? state.pipette.liquids.lowVolumeDefaults.supportedTips[tipType]
+      : state.pipette.liquids.default.supportedTips[tipType]
```

Comment:
i think it is `lowVolumeDefault` instead of `lowVolumeDefaults`? Additionally, we need to make sure the key exists. So maybe for extra protection we can make sure the pipette supports low volume and the key exists in the object.

```suggestion

const liquidSpecs = state.pipette.liquids

  const flowRatesForSupportedTip: SupportedTip | undefined =
    state.volume < 5 && `lowVolumeDefault` in liquidSpecs && LOW_VOLUME_PIPETTES.includes(state.pipette.loadName)
      ? liquidSpecs.lowVolumeDefaults.supportedTips[tipType]
      : liquidSpecs.default.supportedTips[tipType]
```

Reply:
`LOW_VOLUME_PIPETTES` is a constant that can be imported from `shared-data`!

Reply:
This is super helpful, thank you